### DISCUSSION TITLE
Sort by track number

### DISCUSSION THREAD

**Discussion by CykoCyanide17** [PaulWoitaschek/Voice#2295] [Ideas]
I recently added a book that in itself contains multiple "books" and now all my ch 01s are next to each other etc.  They show up in proper order in other apps using their "track number." Wonder if we could make this a feature here?

**Comment by oawd** [comment]
Yes. This sorting option is important. The app should recognize the ID3 tags for disc number and track number.
My audiobooks have a file per chapter and they have the chapter title as file name. They have their disc number and track number in their ID3 data.
When the files are alphabetically sorted by their file names, then the chapters are disordered.

**Reply by abturtle** [reply]
Hey, I'm not the developer but I do have a workaround for you. I use the audio book converter from recoupler.com to combine all of the files into one. M4b file.

**Reply by oawd** [reply]
Thank you for your help.
But my point is that sorting by disc number and track number should be included in the Voice app. One should not have to use a workaround.
